pages/help/help_domain-management.md

```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
 
 Manage DNS information, domain contacts, and more.
  
-- [Manage DNS information for your domain](#)
-- [Add people to your domain account](#)
-- [Add a security email for your domain](#)
-- [Renew your .gov domain](#)
-- [Domain security best practices](#)
+- [Manage DNS information for your domain](#manage-DNS-information-for-your-domain)
```

Comment:
This anchor link isn't working. Looks like the `DNS` portion needs to be lowercase

`#manage-dns-information-for-your-domain`

Reply:
Fixed